velox/exec/Exchange.cpp

```diff
@@ -144,6 +144,14 @@ void Exchange::recordExchangeClientStats() {
   for (const auto& [name, value] : exchangeClient_->stats()) {
     lockedStats->runtimeStats.erase(name);
     lockedStats->runtimeStats.insert({name, value});
+
+    if (name == "backgroundCpuTimeMs") {
+      const CpuWallTiming backgroundTiming{
```

Comment:
CpuWallTiming tracks CPU in nanoseconds, not milliseconds. Do we need to do some conversion?

Reply:
Fixed.